easy_my_coop/models/company.py

```diff
@@ -81,17 +81,12 @@ def _get_base_logo(self):
         help="Text to display aside the checkbox to approve financial risk."
     )
 
-    @api.onchange('internal_rules_approval_required')
-    def onchange_internal_rules_approval_required(self):
-        if self.internal_rules_approval_required:
-            self.display_internal_rules_approval = True
-
-    @api.onchange('data_policy_approval_required')
-    def onchange_data_policy_approval_required(self):
-        if self.data_policy_approval_required:
-            self.display_data_policy_approval = True
-
-    @api.onchange('financial_risk_approval_required')
-    def onchange_financial_risk_approval_required(self):
-        if self.financial_risk_approval_required:
-            self.display_financial_risk_rules_approval = True

```

Comment:
@elouan-lb you can leave @api.onchange method as it act more like helpers than constraint. It auto check the checkbox when the related checkbox is checked. So we should better keep those methods.

Reply:
👍

Reply:
I reverted the changes, thanks for the review.